### PR TITLE
Modified estimateFee method to return both bribe in tokens and in eth

### PR DESCRIPTION
**Pull request by jferas**

### CODE REVIEW

File: zeneth-js/src/fee-estimation.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { getGasPrice, getEthPriceInUsd, getTokenPriceInUsd } from './helpers';
 import { BigNumberish, BigNumber } from '@ethersproject/bignumber';
 import { formatEther, parseUnits } from '@ethersproject/units';
+import { estimateFeeReturn } from './types';
```
Comment (apbendi) [inline zeneth-js/src/fee-estimation.ts:4]: Question for @wildmolasses and @mds1 on TypeScript best practices: would defining the type in a separate file be the typical way this is done? Also does typescript usually capitalize type names?

File: zeneth-js/src/fee-estimation.ts [hunk 2]
```diff
@@ -34,5 +35,5 @@ export const estimateFee = async ({
   const bundlePriceInEth = +formatEther(scaledBundlePriceInWei); // total gas, denominated in ETH
   const bundlePriceInUsd = bundlePriceInEth * ethPrice; // total gas, denominated in dollars
   const tokensNeededForBribe = parseUnits(String(bundlePriceInUsd / tokenPrice), tokenDecimals); // total gas, demoninated in token
-  return tokensNeededForBribe;
+  return { bribeInTokens: tokensNeededForBribe, bribeinEth: bundlePriceInEth };
```
Comment (apbendi) [inline zeneth-js/src/fee-estimation.ts:38]: nit: capitalize`i` in `bribeinEth` => `bribeInEth`

Wondering about the naming for the params as well. Off the top of my head, I wonder if `ethBribe` and `tokensNeeded` or something like that might be clearer? Any thoughts @wildmolasses @mds1?

**Review by apbendi — COMMENTED** [review]
Looks like what we need! Just added some quick comments on naming and typescript convention. Functionality seems right, though!

File: zeneth-js/src/fee-estimation.ts (hunk 1 above)
Comment (wildmolasses) [reply zeneth-js/src/fee-estimation.ts:4]: yes type should be capitalized! Also, I'm not sure we need to define the type separately here. The return type is implied by the function

File: zeneth-js/src/fee-estimation.ts (hunk 2 above)
Comment (wildmolasses) [reply zeneth-js/src/fee-estimation.ts:38]: personally I think `bribeInTokens` and `bribeInEth` expresses the relationship of these numbers better. these aren't two different numbers, these are the same number in different units. Is that correct?

Comment (jferas) [reply zeneth-js/src/fee-estimation.ts:38]: I'm good with those "bribe-based" names

Comment (jferas) [reply zeneth-js/src/fee-estimation.ts:38]: The method is inside a file that itself is called fee-estimation, btw

Comment (apbendi) [reply zeneth-js/src/fee-estimation.ts:38]: I like the suggestions overall @wildmolasses but not sure what `BasedOnBundleCost` is adding here. We pass the bundle gas estimate as an input parameter, so obviously the output is based on the input. `bribeInEth` and `bribeInTokens` seems descriptive enough to me?

Comment (jferas) [reply zeneth-js/src/fee-estimation.ts:38]: Pushing with method name called "estimateFee" and the return fields called "bribeInEth" and "bribeInTokens", just to have something workable sooner.

**Review by apbendi — APPROVED** [review]
Looks good to me 👍

**Comment by wildmolasses** [comment]
lgtm!

**Pull request merged**